src/controllers/submitUrlController.js

```diff
@@ -76,10 +77,11 @@ class SubmitUrlController extends UploadController {
     return url.length <= 2048
   }
 
-  static async getHeadRequest (url) {
+  static async headRequest (url) {
     try {
-      return await axios.head(url)
+      return await axios.head(url, { headers: { 'User-Agent': 'check service' } })
     } catch (err) {
+      logger.info({ message: `SubmitUrlController.headRequest(): err.code=${err.code}`, type: types.External, responseStatus: err?.response?.status, url })
```

Comment:
Should this be `logger.error`?

```suggestion
      logger.error({ message: `SubmitUrlController.headRequest(): err.code=${err.code}`, type: types.External, responseStatus: err?.response?.status, url })
```

Reply:
No. This code is used to check if user supplied URL is accessible. It is expected that that the URL will be incorrect or inaccessible - it's not an 'error' in our application.